servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java

```diff
@@ -119,7 +118,10 @@ default HttpExecutionStrategy requiredOffloads() {
         public DefaultFilterableStreamingHttpLoadBalancedConnection(
                 final FilterableStreamingHttpConnection delegate,
                 final ReservableRequestConcurrencyController controller) {
-            this(delegate, controller, () -> MAX_VALUE);
+            this(delegate, controller, () -> {
+                throw new UnsupportedOperationException("This type of connection doesn't support scoring. " +
```

Comment:
It will be useful to know the type of "this connection", consider:
```patch
Index: servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java b/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java
--- a/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java	(revision f076f7e355de22f777c7c0d304b5c5c2475eee00)
+++ b/servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpLoadBalancerFactory.java	(date 1660162709813)
@@ -119,8 +119,11 @@
                 final FilterableStreamingHttpConnection delegate,
                 final ReservableRequestConcurrencyController controller) {
             this(delegate, controller, () -> {
-                throw new UnsupportedOperationException("This type of connection doesn't support scoring. " +
-                        "Connection scoring is only available through scoring supported load balancers.");
+                throw new UnsupportedOperationException(
+                        DefaultFilterableStreamingHttpLoadBalancedConnection.class.getName() +
+                                " doesn't support scoring. " + ScoreSupplier.class.getName() +
+                                " is only available through " + HttpLoadBalancerFactory.class.getSimpleName() +
+                                " implementations that support scoring.");
             });
         }
 
Index: servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java b/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java
--- a/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java	(revision f076f7e355de22f777c7c0d304b5c5c2475eee00)
+++ b/servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java	(date 1660162856713)
@@ -18,6 +18,7 @@
 import io.servicetalk.client.api.ConnectionFactory;
 import io.servicetalk.client.api.LoadBalancer;
 import io.servicetalk.client.api.LoadBalancerFactory;
+import io.servicetalk.client.api.ScoreSupplier;
 import io.servicetalk.client.api.ServiceDiscovererEvent;
 import io.servicetalk.concurrent.api.Completable;
 import io.servicetalk.concurrent.api.Publisher;
@@ -141,8 +142,11 @@
 
         @Override
         public int score() {
-            throw new UnsupportedOperationException("This type of connection doesn't support scoring. " +
-                    "Connection scoring is only available through scoring supported load balancers.");
+            throw new UnsupportedOperationException(
+                    DefaultFilterableStreamingHttpLoadBalancedConnection.class.getName() +
+                            " doesn't support scoring. " + ScoreSupplier.class.getName() +
+                            " is only available through " + HttpLoadBalancerFactory.class.getSimpleName() +
+                            " implementations that support scoring.");
         }
 
         @Override

```